vaadin-flow-components-shared-parent/vaadin-flow-components-base/src/main/resources/META-INF/resources/frontend/tooltip.ts

```diff
@@ -3,9 +3,19 @@ import { Tooltip } from '@vaadin/tooltip/src/vaadin-tooltip.js';
 const _window = window as any;
 _window.Vaadin = _window.Vaadin || {};
 _window.Vaadin.Flow = _window.Vaadin.Flow || {};
+_window.Vaadin.Flow.tooltip = _window.Vaadin.Flow.tooltip || {};
 
-_window.Vaadin.Flow.tooltip = {
-  setDefaultHideDelay: (hideDelay: number) => Tooltip.setDefaultHideDelay(hideDelay),
-  setDefaultFocusDelay: (focusDelay: number) => Tooltip.setDefaultFocusDelay(focusDelay),
-  setDefaultHoverDelay: (hoverDelay: number) => Tooltip.setDefaultHoverDelay(hoverDelay),
+_window.Vaadin.Flow.tooltip.setDefaultHideDelay = (hideDelay: number) => Tooltip.setDefaultHideDelay(hideDelay);
+_window.Vaadin.Flow.tooltip.setDefaultFocusDelay = (focusDelay: number) => Tooltip.setDefaultFocusDelay(focusDelay);
+_window.Vaadin.Flow.tooltip.setDefaultHoverDelay = (hoverDelay: number) => Tooltip.setDefaultHoverDelay(hoverDelay);
```

Comment:
This could be shortened to:
```js
const tooltip = (((window as any).Vaadin ||= {}).Flow ||= {}).tooltip ||= {};

Object.assign(tooltip, {
  setDefaultHideDelay: (hideDelay: number) => Tooltip.setDefaultHideDelay(hideDelay),
  setDefaultFocusDelay: (focusDelay: number) => Tooltip.setDefaultFocusDelay(focusDelay),
  setDefaultHoverDelay: (hoverDelay: number) => Tooltip.setDefaultHoverDelay(hoverDelay),  
});
```

Reply:
Thanks, updated `tooltip.ts`. Also updated `executeJS` to use `||=` and `?` .